apps/digibyteprice/digibyteprice.go

```diff
@@ -17,7 +17,7 @@ func New() manifest.Manifest {
 		Name:        "DigiByte Price",
 		Author:      "Olly Stedall @saltedlolly",
 		Summary:     "Display DigiByte Price",
-		Desc:        "Displays the current DigiByte price in one or two fiat currencies and/or in Satoshis. Price data is obtained from Coingecko and updated every 15 minutes. If you would like an additional currency supported, pease let me know in the Tidbyt community Discord.",
+		Desc:        "Displays the current DigiByte price in one or two fiat currencies and/or in Satoshis. Data provided by CoinGecko. Updated every 5 minutes. If you would like an additional currency supported, pease let me know in the Tidbyt community Discord.",
```

Comment:
```suggestion
		Desc:        "Displays the current DigiByte price in one or two fiat currencies and/or in Satoshis. Data provided by CoinGecko. Updated every 10 minutes. If you would like an additional currency supported, pease let me know in the Tidbyt community Discord.",
```

Reply:
Oops. Sorry, I kept changing my mind on how often I should refresh. 10 minutes is fine.